Dockerfile

```diff
@@ -1,36 +1,28 @@
-# NOTE: We do not use this version of Luigi, because a Docker image isn't
-# available for our desired version. `environment.yml` specifies the actual
-# version we use.
-FROM axiom/docker-luigi:2.8.13
-# TODO: Build image from miniforge/minimamba
+FROM axiom/docker-luigi:2.8.13-alpine AS luigi
+# This build stage only exists to grab files
 
-RUN conda install mamba~=0.15.2
+FROM mambaorg/micromamba:0.17.0 AS micromamba
+COPY --from=luigi /bin/run /usr/local/bin/luigid
+USER root
 
-# `libgl1-mesa-glx` is required for qgis to work
-# `unrar` is required for layer pipelines
+# `libgl1-mesa-glx` is required for pyqgis
 # `git` is required for analyzing the current version
-RUN apt-get update && \
-  apt-get install -y \
-  libgl1-mesa-glx \
-  unrar \
-  git
+RUN apt-get update && apt-get install -y \
+  git \
+  libgl1-mesa-glx
 
-# TODO install to `qgreenland` specific environment. Activate in Dockerfile if
-# possible.
-COPY environment-lock.yml .
-RUN mamba env update --file environment-lock.yml --name base
+# Create environments
+RUN micromamba install -y -c conda-forge -n base conda mamba
 
-# Create a dedicated env for shelling out commands
-COPY environment.cmd.yml .
-RUN mamba env create --file environment.cmd.yml
+COPY --chown=micromamba:micromamba environment-lock.yml /tmp/environment.yml
+RUN micromamba install -y -n base -f /tmp/environment.yml
```

Comment:
Install our environment to the base env with no worries about miniconda conflicts. However, `conda` is still in the base env.